src/push-notifications.test.js

```diff
@@ -549,11 +549,102 @@ describe('interest methods', () => {
   });
 });
 
+describe('.getState', () => {
+  const instanceId = 'df3c1965-e870-4bd6-8d75-fea56b26335f';
+
+  describe('if SDK is started', () => {
+    let devicestatestore = require('./device-state-store');
+    beforeEach(() => {
+      devicestatestore.default = makeDeviceStateStore({
```

Comment:
We've repeated this device state store stuff enough at this point, it's probably worth pulling out into a fixture/helper. Something like `setUpDeviceState({....})`

Reply:
I don't think it's too bad as it is currently. The `makeDeviceStateStore` helper does most of the work. I'm not going to make any changes now unless you feel strongly about it.